### ISSUE TITLE
"Select untouched" button also selects developed pictures

### ISSUE THREAD

**Issue by cisoun** [darktable-org/darktable#12772]
> **Is your feature request related to a problem? Please describe.**

On the lighttable, when clicking on the "Select untouched" button (not sur how it is translated in english), developed pictures are also selected. Shouldn't it be only the ones without set parameters (or at least XMP files assigned)?

> **Describe the solution you'd like**

Select all pictures that are left untouched (no XMP file assigned).

> **Alternatives**

Manual selection.

> **Additional context**

Darktable 4.0.1+1~ga59e0af56
Mac OS

Thanks for your help!

**Comment by Nilvus** [comment]
This button, since a long long time, is just to select images that are not been manually processed. It so select images that have been processed by automatic presets on import, not the other ones. 
On a collection where I have manually processed all my images and if I click on the button, 0 images is selected. I use actual master but this part had not been changed since a time.

If I go on a collection where I processed only a few images, this button select all other. It so works as expected.

This button will select all images without the "development" icon on thumbnails in lighttable. See screenshot below where the first 2 images are processed (development icon is on top right) and the 2 other without this icon and selected after clicking on select untouched button):

![image](https://user-images.githubusercontent.com/45535283/200113679-71b07a68-3a05-4994-8f2c-2f886fe31fe6.png)
